test/steward_owed.js

```diff
@@ -421,8 +425,8 @@ contract("WildcardSteward owed", (accounts) => {
 
     const depositCalc = secondBuyValue.sub(price);
 
-    expectEvent.inLogs(logs, "Foreclosure", { prevOwner: accounts[2] });
-    expectEvent.inLogs(logs, "Buy", { owner: accounts[3], price: ether("2") });
+    expectEvent(txReceipt, "Foreclosure", { prevOwner: accounts[2] });
```

Comment:
We get a warning that the `expectEvent.inLogs` method is being deprecated. So I updated it.